### ISSUE TITLE
Error: Byte Order: Expected feff saw 0000

### ISSUE THREAD

**Issue by alfaz151** [sakai-to/node-red-contrib-spreadsheet-in#18]
@sakai-to 
## Problem
When attempting to read an old format XLS file (pre-2007) using the book node from the `node-red-contrib-spreadsheet-in` package, an error is thrown: "Byte Order: Expected feff saw 0000."
## Solution
To resolve the "Byte Order: Expected feff saw 0000" error when reading old format XLS files with the `book` node from `node-red-contrib-spreadsheet-in`, modify the code of book node as follows:
```javascript
var option = {
    type: typeof msg.payload === "string" ? "string" : "buffer",
    raw: config.raw,
    // dateNF: "yyyy-mm-dd",
    password: undefined,
    cellNF: true,
    cellStyles: true,
    WTF: false, // set to false to resolve byte order error.
};
msg.payload = XLSX.read(msg.payload, option);
```
This change sets the WTF option to false, resolving the byte order error.

**Comment by sakai-to** [comment]
@alfaz151 Thank you for the suggestion. I added a test case for the `book` node and confirmed that the error was resolved by your solution. I will release a new version including this fix soon.

**Comment by sakai-to** [comment]
I have released the new version 0.7.2, including the fix for this issue.
